### DISCUSSION TITLE
E/ConnectivityManager.CallbackHandler: callback not found for RELEASED message

### DISCUSSION THREAD

**Discussion by jlriverob** [firebase/firebase-android-sdk#4047] [Q&A] [answered]
Buenos dias,

Soy un poco nuevo con esto de github.

Quisiera un apoyo de parte de ustedes,

Hace un tiempo comence dos desarrollos que se conectaran a firebase, al principio funcionaron perfectamente hasta hace unos dias donde la maquina donde estoy programando comenzo a darme este error:

E/ConnectivityManager.CallbackHandler: callback not found for RELEASED message

luego cuando intenta escribir en firebase me lanza este otro error:
W/Firestore: (24.2.2) [WriteStream]: (c3d9f49) Stream closed with status: Status{code=UNAVAILABLE, description=Channel shutdownNow invoked, cause=null}.

ya he agotado todos mis conocimientos, por lo que recurro a esta comunidad a ver si a alguien le ha dado el mismo error y como solucionarlo.

En mi caso el programa correo cuando uso el dispositivo movil como emulador pero el problema sucede cuando lo ejecuto en el emulador de Android, no estoy usando flutter para que tengan conocimiento de como se esta ejecutando la aplicacion.

Saliudos y espero sus comentarios a ver si logro solventar este problema.

Gracias

**Comment by daymxn** [comment, marked as answer]
Me encantaría ayudarte:)

Para empezar, quiero señalar que esto se publicó en nuestro foro de discussion, y debería haberse publicado en nuestra página de issues. Puede encontrar nuestra página de issues [aquí](https://github.com/firebase/firebase-android-sdk/issues).

En cuanto a tu problema, necesitaríamos ver el code que está causando el error para ofrecer mucha ayuda. Aunque parece que el emulador no tiene acceso a Internet.

Voy a cerrar esto como respondido, y si usted todavía está teniendo este problema no dude en hacer un post en nuestra página de issues (que he enumerado anteriormente).
